### PR TITLE
Fix typos

### LINKED ISSUE #10: Typos in various packet names
There are typos in various packets' names.

### PR DESCRIPTION
**Pull request by lzardy**
Will be required for a future PR for GameServer related to movement.

Resolve #10

### CODE REVIEW

File: LeaguePackets/GamePacketID.cs [hunk 1]
```diff
@@ -166,7 +166,7 @@ public enum GamePacketID : uint
         S2C_ToggleInputLockFlag = 0x9B,                 //DONE?
         C2S_SoftReconnect = 0x9C,                       //DONE?
         S2C_CreateTurret = 0x9D,                        //DONE?
-        NPC_Die_Broadcast = 0x9E,                                 //DONE?
+        NPC_Die_Broadcast = 0x9E,                        //DONE?
```
Comment (danil179) [inline LeaguePackets/GamePacketID.cs]: There is additional space here.

**Review by danil179 — APPROVED** [review]
Approved. If you have time to fix this single space it would be great.

File: LeaguePackets/GamePacketID.cs (hunk 1 above)
Comment (danil179) [reply LeaguePackets/GamePacketID.cs]: I'll merge this and open an issue about this single space before the `//` comment.

Comment (lzardy) [reply LeaguePackets/GamePacketID.cs]: Fixed.

**Pull request merged**